agent.go

```diff
@@ -198,8 +198,8 @@ func (s *sandbox) getRunningProcess(cid, execID string) (*process, *container, e
 		return nil, nil, err
 	}
 
-	if status != libcontainer.Running {
-		return nil, nil, fmt.Errorf("Container %s %s, should be %s", cid, status.String(), libcontainer.Running.String())
+	if status == libcontainer.Stopped {
```

Comment:
We may want to rename `getRunningProcess` to `getCreateProcess` or simply `getProcess` then.

Reply:
Done.